src/components/subscriptions/styles/LicenseStatus.scss

```diff
@@ -0,0 +1,9 @@
+@import "~@edx/paragon/scss/edx/utilities-only";
+
+.table-striped {
+  td {
+    p {
+      color:#ec90f5
```

Comment:
This is magenta; it should be a light gray (try $gray-300 or $gray-500, which ever is more legible). Related, would it better to target a class name, e.g. `.pending-since-date`, instead of the table elements?

nit: space between `color:` and the property value. Also, a semicolon is needed :)

Reply:
@adamstankiewicz I have followed the mockups. In the mockups it is magenta. I am attaching the screenshot for clarification.
<img width="620" alt="Screenshot 2020-06-25 at 7 17 23 PM" src="https://user-images.githubusercontent.com/11922730/85737405-cc207a00-b718-11ea-8b33-907da07ac14d.png">

Reply:
Magenta text is Brandon C's way of calling out specific things in a mockup, but it is not the color the text should actually be... A magenta does not align with the color scheme on this page; it should be a gray, like the `.text-muted` class will do.

Reply:
What is your suggestion? Should I stick with magenta or gray?

Reply:
Gray, using the `.text-muted` class.